ansible/roles/omero-python-deps/README.md

```diff
@@ -28,9 +28,28 @@ Role Variables
 --------------
 
 Optional variables:
-- `omero_python_deps_profile`: Either `distribution` to use the distribution packages for Python modules where possible (default), or `pypi` to install recommended packages globally from PyPI using `pip`.
 - `omero_python_deps_recommended`: If `False` install a minimal set of Python dependencies, default `True`.
 
+If `omero_python_deps_recommended` False then use virtualenv role
```

Comment:
@manics are you ok with removal of omero_python_deps_profile and handling that as described below?